app/models/measure.rb

```diff
@@ -173,6 +173,12 @@ def additional_code_sort
     end
   end
 
+  def measure_conditions_with_valid_guidance
+    measure_conditions.select do |mc|
+      mc.document_code.present? && (mc.guidance_cds.present? || mc.guidance_chief.present?)
```

Comment:
You could make the measure condition model responsible for knowing about what it means to have guidance:

```ruby
measure_conditions.select(&:has_guidance?)
```